jboss-tck-runner/pom.xml

```diff
@@ -236,17 +247,46 @@
                 <plugins>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-surefire-plugin</artifactId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <version>3.1.2</version>
```

Comment:
This is also inherited from weld-parent POM and thereby redundant.
See https://github.com/weld/parent/blob/master/pom.xml#L90